src/components/common/Spinner.js

```diff
@@ -0,0 +1,15 @@
+import React, { Fragment } from 'react';
+import spinner from './spinner.gif';
+const Spinner = () => {
+  return (
+    <Fragment>
+      <img
+        src={spinner}
+        style={{ width: '200px', margin: 'auto', display: 'block' }}
+        alt="Loading..."
+      />
+    </Fragment>
```

Comment:
NB: You can add a fragment like this too. It will save the unnecessary import.
```suggestion
    <>
      <img
        src={spinner}
        style={{ width: '200px', margin: 'auto', display: 'block' }}
        alt="Loading..."
      />
    </>
```